.github/ISSUE_TEMPLATE.md

```diff
@@ -7,4 +7,4 @@
 ### Steps to reproduce the behavior
 
 
-##### Tested on [device], iOS [version], WPiOS [version]
+##### Tested on [device], iOS [version], JPiOS / WPiOS [version]
```

Comment:
What do you think of using full names for the apps? We might have gotten away with shortening WordPress to WP because the context of the repo made it self evident what we meant, but I do wonder if outside contributors would immediately recognize "JP" as a shortcut for Jetpack.

```suggestion
##### Tested on [device], iOS [version], Jetpack iOS / WordPress iOS [version]
```

Reply:
SGTM!